From 5e3cdea82bd5870af28132b9ef87288d58482527 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 06 Jan 2019 17:06:20 +0200
Subject: [PATCH] [SW-2939] pll: do not initialize the CPU PLL

Remove the CPU PLL initialization from preboot, as a w/a to crashes
that are seen if we change the ARM core frequency using the
PLL clock switch.
Now the CPU PLL is initialized in BTL.

Change-Id: I6f716eccbad51e736bf5eb1df737b7b6f78d9ff6
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/pll.c b/pll.c
index dfc8fff..3771ba6 100644
--- a/pll.c
+++ b/pll.c
@@ -202,7 +202,7 @@
 	struct hl_pll_cfg *hl_pll_cfg;
 
 	pll_num = hl_pll_get_cfg(&hl_pll_cfg);
-	cpu_pll_bypassed = hl_pll_set(hl_pll_cfg);
+
 	for (i=1; i<pll_num; i++)
 		hl_pll_set(hl_pll_cfg + i);
 }
